tests/docker/test.sh

```diff
@@ -27,6 +27,8 @@ fi
 SHORTEN_LENGTH=${#EXPECTED_SHORTEN_CONTAINER_ID} # the length of the short format
 ACTUAL_SHORTEN_CONTAINER_ID="${ACTUAL_CONTAINER_ID:0:$SHORTEN_LENGTH}"
 
+docker stop $ACTUAL_CONTAINER_ID
```

Comment:
Nothing to do with actual PR but I notices we weren't stopping the container after the test.